prog/vnet/cert_nexus.rb

```diff
@@ -101,6 +101,17 @@ def before_run
     nap 60 * 60 * 24 * 30 # 1 month
   end
 
+  label def restart
+    when_restarted_set? do
+      decr_restarted
+      update_stack_restart_counter
+      hop_start
+    end
+
+    cert.incr_restarted
+    nap [60 * (strand.stack.first[:restarted] + 1), 60 * 10].min
```

Comment:
nit:
```suggestion
    nap [60 * (frame["restarted"] + 1), 60 * 10].min
```

Reply:
Also, if you don't update the stack of the existing strands, it will fail. Please don't forget to update them.